guidelines/taxation.md

```diff
@@ -0,0 +1,41 @@
+# Taxation
```

Comment:
Not sure this belongs to guidelines. Guidelines are meant as technical guidelines if you check out other content there.